app.js

```diff
@@ -25,12 +26,13 @@ app.all('/*', (request, response, next) => {
 // register error handling middleware: pSQL errors
 app.use((err, request, response, next) => {
 	/**
+	 * Note (From Feedback): currently only require 22P02, add other codes in as they become required
 	 * 22P02: invalid text representation
 	 * 42601: syntax error
 	 * 42703: undefined column or parameter (limit is cast to NaN << throws this code)
 	 * 23502: violates not null constraint
 	 */
-	const badRequestErrCodes = ['22P02', '42601', '42703', '23502'];
+	const badRequestErrCodes = ['22P02'];
 	if (badRequestErrCodes.includes(err.code)) {
```

Comment:
This is a nice way of doing it! Not seen it done with an array that often. Great way of dealing with more and more error codes